bin/templates/cordova/lib/prepare.js

```diff
@@ -43,8 +43,16 @@ module.exports.prepare = function (cordovaProject, options) {
 
     this._config = updateConfigFilesFrom(cordovaProject.projectConfig, munger, this.locations);
 
+    // Prepare the min and target SDK version
+    const minSdkVersion = this._config.getPreference('android-minSdkVersion', 'android');
+    const targetSdkVersion = this._config.getPreference('android-targetSdkVersion', 'android');
+
+    let gradlePropertiesUserConfig = {};
+    if (minSdkVersion) gradlePropertiesUserConfig.minSdkVersion = minSdkVersion;
```

Comment:
Please confirm if `gradlePropertiesUserConfig.minSdkVersion` is the correct property name to be setting or if it should be prefixed with `cdv` such as `gradlePropertiesUserConfig.cdvMinSdkVersion`.

In my example code, I used `minSdkVersion` because it was reference in [Google Android Developer Docs](https://developer.android.com/studio/publish/versioning#minsdkversion).

If you looked at @dpogue commit, he used the prefix version. https://github.com/dpogue/cordova-android/commit/d18f8d18a10a50084130300d5edbb84901b59707

Reply:
I believe we want the `cdv`-prefixed versions, because that's what [build.gradle is looking for](https://github.com/apache/cordova-android/blob/master/bin/templates/project/app/build.gradle#L46-L90)

Reply:
Agreed on my part to use cdv prefix

Reply:
now done